setup.py

```diff
@@ -16,23 +16,24 @@
     version=VERSION,
     url="https://github.com/Harvard-University-iCommons/django-canvas-oauth",
     author="Jaime Bermudez",
-    author_email="tlt-ops@g.harvard.edu, jaime_bermudez@harvard.edu",
+    author_email="jaime_bermudez@harvard.edu",
```

Comment:
Are we sure we don't have a better group email, like `academictechnology@harvard.edu`?

Reply:
Good question, I'm not aware if we have a better group email, but I will check with some folks to see if there is.

Reply:
Yes, you're right, looks like that's what we should be using based on some of our other projects. Thanks for the tip!